.ci/scripts/install-test-dependencies.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+set -euxo pipefail
+#
+# Install the dependencies using in tests.
```

Comment:
Should we find a way to bail out of this script or return an error code if all the dependencies were not installed successfully?

Reply:
In last commit I address this: instead of checking the tools when the Golang code is initialised, it's done as part of the BeforeScenario of the test suite. Then the test, if executed, will fail because the tools are not there.

In the other hand, we must install the tools in the workers, using Infra scripts. Wdyt?

Reply:
> In the other hand, we must install the tools in the workers, using Infra scripts. Wdyt?

This seems like a reasonable compromise to me. I like the idea of doing a pre-flight check in the `BeforeScenario`.